connection.py

```diff
@@ -165,15 +165,70 @@ def insert_data(db):
                 cursor.execute(query_calificaciones)
                 db.commit()
 
+                print("Datos insertados!")
+        else:
+            print("No hay tablas aún!")
     except:
         db.rollback()
         print(">>Error al insertar contenido!")
-        print(sys.exc_info()[0])
-    else:
-        print("Datos insertados!")
+        print(sys.exc_info()[0]) 
+
+def del_data(db):
+    try:
+        cursor = db.cursor()
+        tablesDatabase = get_tables(db)
+        if(len(tablesDatabase) != 0):
+            opt = int(input("Inserte un número\n\n1-Borrar todos los datos\t2-Borrar datos y tablas\t\t0-Cancelar\n\nOpción: "))
+
+            if(opt == 0):
+                print("\n.:Cancelar:.\nNingún registro fue afectado")
+            elif(opt == 1):
+                print("\n.:Borrar todos los datos:.")
 
-def del_data():
-    pass
+                confirmation = int(input('¡Advertencia, se borrarán todos los datos!\n¿Continuar? (0 - No\t 1 - Sí)\nOpción: '))
+                if(confirmation == 0):
+                    print("Abortando borrado de registros!")
+                elif(confirmation == 1):
+                    print("\nBorrando datos...")
+
+                    drop_constraints(db)
+                    query_calificaciones = ("TRUNCATE TABLE Calificaciones;")
+                    query_materia = ("TRUNCATE TABLE Materia;")
+                    query_alumno = ("TRUNCATE TABLE Alumno;")
+
+                    datos_borrados = count_records(db,'Alumno')
+
+                    cursor.execute(query_calificaciones)
+                    cursor.execute(query_materia)
+                    cursor.execute(query_alumno)
+
+                    print("Se borraron {0}!".format(datos_borrados))
+                    put_constraints(db)
+                else:
+                    print("Opción no válida, abortando borrado de registros!")
+            elif(opt == 2):
+                print("\n.:Borrar datos y tablas:.")
+
+                confirmation = int(input('¡Advertencia, se borrarán todos los datos y las tablas también!\n¿Continuar? (0 - No\t 1 - Sí)\nOpción: '))
+                if(confirmation == 0):
+                    print("Abortando borrado de registros!")
+                elif(confirmation == 1):
+                    print("\nBorrando datos...")
+                    
+                    datos_borrados = count_records(db,'Alumno')
```

Comment:
I would put variable name 'deleted_data'.
```suggestion
                    deleted_data = count_records(db,'Alumno')
```